src/AntTweakBar/TwMgr.cpp

```diff
@@ -6352,6 +6352,8 @@ void CTwMgr::SetCursor(CTwMgr::CCursor _Cursor)
         if( ok && (ci.flags & CURSOR_SHOWING) )
             ::SetCursor(_Cursor);
     }
+#else
+	(void)_Cursor;
```

Comment:
Not so important in this specific case as it is part of AntTweakBar's source code (which will eventually be deprecated and removed), but I think in cases like this we want to use some sort of `#pragma` or macro to declare that the variable is unused. Otherwise the code looks a bit mysterious.